services/apps/alcs/src/portal/public/search/public-search.controller.ts

```diff
@@ -246,7 +249,7 @@ export class PublicSearchController {
       localGovernmentName: notification.localGovernmentName,
       ownerName: notification.applicant,
       class: 'NOTI',
-      status: notification.status.status_type_code,
+      status: notification.status.status_type_code
```

Comment:
This needs a prettier format I think. Looks like on some files your code is using a trailing comma but not all of them. For some reason on this file it looks like it IS NOT adding trailing commas.
  "trailingComma": "all" is the prettier setting in the .prettierrc